.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+venv
```

Comment:
Add EOF(End-of-File), need to add an empty line at the end of the file
![image](https://github.com/mate-academy/rda_task_1_create_database/assets/87720583/350c0f34-a6fa-47e9-a3f4-d922a5593e39)